selenium/firefox/selenium/entrypoint.sh

```diff
@@ -1,4 +1,19 @@
 #!/bin/bash
 SCREEN_RESOLUTION=${SCREEN_RESOLUTION:-"1920x1080x24"}
 DISPLAY=99
-/usr/bin/xvfb-run -l -n "$DISPLAY" -s "-ac -screen 0 $SCREEN_RESOLUTION -noreset -listen tcp" /usr/bin/java -Xmx256m -Djava.security.egd=file:/dev/./urandom -jar /usr/share/selenium/selenium-server-standalone.jar -port 4444 -browserTimeout 120
\ No newline at end of file
+/usr/bin/xvfb-run -l -n "$DISPLAY" -s "-ac -screen 0 $SCREEN_RESOLUTION -noreset -listen tcp" /usr/bin/java -Xmx256m -Djava.security.egd=file:/dev/./urandom -jar /usr/share/selenium/selenium-server-standalone.jar -port 4444 -browserTimeout 120 &
+XVFB_PID=$!
+
+for i in $(seq 1 10)
```

Comment:
How about using `until` here? E.g. like this: https://github.com/aerokube/selenoid-images/blob/master/selenium/video/entrypoint.sh#L9

Reply:
Yes, good idea. I have added a new patch with that.